frontend/src/routes/_oh.tsx

```diff
@@ -313,11 +303,9 @@ export default function MainApp() {
             <p className="text-xs text-[#A3A3A3]">
               To continue, connect an OpenAI, Anthropic, or other LLM account
             </p>
-            {isConnected && (
-              <p className="text-xs text-danger">
-                Changing settings during an active session will end the session
-              </p>
-            )}
+            <p className="text-xs text-danger">
+              Changing settings during an active session will end the session
+            </p>
```

Comment:
Note that the warning will always be rendered

<img width="479" alt="image" src="https://github.com/user-attachments/assets/ea3856e3-0c94-470e-b82d-afe2002b25ee">

Reply:
This actually may be annoying. With these changes we're either always going to show this error or never since the parent does not have access to the context. 🤔